client/state/jetpack-onboarding/reducer.js

```diff
@@ -16,6 +19,20 @@ export const credentialsReducer = keyedReducer(
 	)
 );
 
+export const settingsReducer = keyedReducer(
```

Comment:
FWIW, personally, I like to drop the `Reducer` part from the name.

Reply:
Sure, but then we wouldn't be able to take advantage of using shorthands in `{ settings }` and `{ credentials }`, so it's just a compromise and a choice we're going with here.

Reply:
Oh, the actions destructuring? Okay, makes sense. (I mean we could alias it there -- `{ settings: settingsState }` -- but that isn't exactly pretty either. Fine to stick with the `...Reducer` suffices.)